src/index.js

```diff
@@ -1,13 +1,12 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import './index.css';
-import App from './App';
+import Calculator from './Components/Calculator';
 import reportWebVitals from './reportWebVitals';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <App />
+    <Calculator />
   </React.StrictMode>,
 );
 
```

Comment:
- [ ] Nice work so far 👍🏿 but as for best practices you are expected to import your component into the App.js not directly importing your component into the index.js file